webapp/channels/src/actions/notification_actions.jsx

```diff
@@ -312,24 +313,7 @@ export function sendDesktopNotification(post, msgProps) {
 export const notifyMe = (title, body, channel, teamId, silent, soundName, url) => (dispatch) => {
     // handle notifications in desktop app
     if (isDesktopApp()) {
-        const msg = {
-            title,
-            body,
-            channel,
-            teamId,
-            silent,
-        };
-        msg.data = {soundName};
-        msg.url = url;
-
-        // get the desktop app to trigger the notification
-        window.postMessage(
-            {
-                type: 'dispatch-notification',
-                message: msg,
-            },
-            window.location.origin,
-        );
+        dispatchNotification(title, body, channel.id, teamId, silent, soundName, url);
```

Comment:
I'd be fine making sure the parameters are consistently ordered, but again other than protecting people from fat-fingering I don't see any value. I'd much rather just be prescriptive about the API and what we add, and I'd like to avoid having an object that could be misused.

Again, 3/5 on that.

Reply:
I'd say preventing mistakes is pretty valuable, but I'm fine to leave this